packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -3,6 +3,53 @@
     display: flex;
     flex-direction: column;
 
+    &__dropdown {
+        box-shadow: 0 1rem 2rem var(--shadow-menu);
+        border-radius: $BORDER_RADIUS;
+        cursor: pointer;
+        position: absolute;
+        right: 0;
+
+        span {
+            color: var(--text-prominent) !important;
```

Comment:
Can you find a workaround for these `!important`s? They're definitely not preferred. Use the Text component instead?

Reply:
managed to get a workaround with Text Component  👍